db/migrate/20160421102740_update_travel_advice_backend_id.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateTravelAdviceBackendId < Mongoid::Migration
+  def self.up
+    Route.where(backend_id: "frontend", incoming_path: /^\/foreign-travel-advice\/[aA-zZ-]+$/).each do |route|
```

Comment:
Could this use `format: 'travel_advice'` instead of the regex on the path? It's more explicit.